### ISSUE TITLE
Branch protection?

### ISSUE THREAD

**Issue by pancetta** [Parallel-in-Time/qmat#7]
It may be worth considering to protect the main branch from force pushing or deletion, or require status checks before merging. See https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets for details.

**Comment by tlunet** [comment]
Yes, this is planned to be done before going to version 1.*.*, which will be dump once the pipeline for version update is fully defined and documented.

Currently still exploring the different solutions in order to find de most easiest and safest one.
